src/main/java/org/junitpioneer/jupiter/cartesian/ArgumentSets.java

```diff
@@ -149,7 +149,7 @@ public final <T> ArgumentSets argumentsForNextParameter(Stream<T> arguments) {
 	}
 
 	List<List<?>> getArguments() {
-		return argumentSets;
+		return parameterSets;
```

Comment:
I find `parameterSets` within `ArgumentSets` a bit confusing. Alternatively, you could use the name of this getter, i.e., name the field `arguments`.

Reply:
👍